src/lib/SupabaseRealtimeClient.ts

```diff
@@ -66,13 +62,30 @@ export class SupabaseRealtimeClient {
    * Enables the channel.
    */
   subscribe(callback: Function = () => {}) {
+    // if the socket already has a good accessToken
+    // we can just use it strait away∏
```

Comment:
```suggestion
    // we can just use it straight away
```

Reply:
@alaister did you intend to do `∏`? what even is that?

Reply:
Nice catch! It looks like I've been button-mashing again...